### PR TITLE
Introduction to Grid lesson: Fix grammer mistake in subheading

### PR DESCRIPTION
**Pull request by arpit-srivastava-one**
Fixed a small grammer mistake in one of the subheadings -

### Earlier
```
What the Grid?
```

### Updated
```
What is Grid?
```

### CODE REVIEW

**Review by DanatGutema — COMMENTED** [review]
Although Grid is a newer module to CSS, this layout tool has been long in development

**Comment by arpit-srivastava-one** [comment]
> Although Grid is a newer module to CSS, this layout tool has been long in development

Thanks for pointing this out. I've did the change and made the commit.

File: intermediate_html_css/grid/introduction_to_grid.md [hunk 1]
```diff
@@ -54,9 +54,9 @@ But setting up a two-dimensional layout of cards would be much simpler using CSS
 </p>
 <script async src="https://cpwebassets.codepen.io/assets/embed/ei.js"></script>
 
-### What the Grid?
+### What is Grid?
 
-Although Grid is a newer module to CSS, this layout tool has long been in development. Fun fact, CSS co-creator Dr. Bert Bos (no relation to Wes Bos) started working on this layout model in 1996. The idea was inspired by the use of grid-like layouts in other forms of media like newspapers and magazines. After years of thought demonstrations and development, CSS Grid was finally introduced to all major browsers in 2017.
+Although Grid is a newer module to CSS, this layout tool has been long in development. Fun fact, CSS co-creator Dr. Bert Bos (no relation to Wes Bos) started working on this layout model in 1996. The idea was inspired by the use of grid-like layouts in other forms of media like newspapers and magazines. After years of thought demonstrations and development, CSS Grid was finally introduced to all major browsers in 2017.
```
Comment (zachmmeyer) [inline intermediate_html_css/grid/introduction_to_grid.md:59]:
```suggestion
Although Grid is a newer module to CSS, this layout tool has long been in development. Fun fact, CSS co-creator Dr. Bert Bos (no relation to Wes Bos) started working on this layout model in 1996. The idea was inspired by the use of grid-like layouts in other forms of media like newspapers and magazines. After years of thought demonstrations and development, CSS Grid was finally introduced to all major browsers in 2017.
```

File: intermediate_html_css/grid/introduction_to_grid.md [hunk 2]
```diff
@@ -54,9 +54,9 @@ But setting up a two-dimensional layout of cards would be much simpler using CSS
 </p>
 <script async src="https://cpwebassets.codepen.io/assets/embed/ei.js"></script>
 
-### What the Grid?
+### What is Grid?
```
Comment (zachmmeyer) [inline intermediate_html_css/grid/introduction_to_grid.md:57]:
```suggestion
### What's the Grid?
```

**Review by zachmmeyer — CHANGES_REQUESTED** [review]
Hello! Thanks for offering to contribute to the curriculum! I've got some changes I'd like to request to your PR before we merge it.

**Review by DanatGutema — COMMENTED** [review]
Which CSS layout module would you use to easily make equal sized items in a container
should replaced by
Which CSS layout module used to easily make equal sized items in a container

File: intermediate_html_css/grid/introduction_to_grid.md (hunk 2 above)
Comment (arpit-srivastava-one) [reply intermediate_html_css/grid/introduction_to_grid.md:57]: Isn't ```What's the Grid?``` grammatically incorrect?

Comment (zachmmeyer) [reply intermediate_html_css/grid/introduction_to_grid.md:57]: Oh yeah you're right, my bad.

**Review by zachmmeyer — APPROVED** [review]
There we go. Thanks for contributing, your patience and dealing with me!

**Pull request merged**